<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QWidget" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Calibration parameters</string>
  </property>
  <widget class="QGroupBox" name="groupBox_CalibSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>441</width>
     <height>481</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0px 4px 0 4px;
}

</string>
   </property>
   <property name="title">
    <string>Calibration settings</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_NrCols">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_NrRows">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_MapName">
      <property name="text">
       <string>Name of ouput map file</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_LogName">
      <property name="text">
       <string>Name of ouput log file</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_InputType"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_ImgList">
      <property name="text">
       <string>Image list file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_CalibPattern"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_NrFrames">
      <property name="text">
       <string>24</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_ImgList">
      <property name="text">
       <string>/home/arjun/workspace/PanoGen/resources/calibration_input/image_list.xml</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_NrRows">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string># of Rows</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_CalibPattern">
      <property name="text">
       <string>Calibration pattern</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_MapName">
      <property name="text">
       <string>CalibOut_Map.xml</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_NrFrames">
      <property name="text">
       <string># of Frames</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_LogName">
      <property name="text">
       <string>ClibOut_CameraData.xml</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_ShowUndistort">
      <property name="text">
       <string>Show the undistorted images</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_ChooseImgList">
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_InputType">
      <property name="text">
       <string>Input type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_NrCols">
      <property name="text">
       <string># of Columns</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_SaveUndistort">
      <property name="text">
       <string>Save the undistorted images</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_Calibrate">
      <property name="text">
       <string>Find Calibration parameters</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Image">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>841</width>
     <height>471</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ImageTitle">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>15</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detected chessboard pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ImageCounter">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>520</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>24/24</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CropImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
